src/components/TodoContent/TodoMain.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC, useContext } from 'react';
+import { TodoInfo } from './TodoInfo';
+import { TodoContext } from '../TodoContext';
+
+export const TodoMain: FC = () => {
+  const { todos } = useContext(TodoContext);
+
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => (
+        <TodoInfo todo={todo} key={todo.id} />
+      ))}
+    </section>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings

Reply:
still not fixed
<img width="682" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/70220667/6378f700-9caf-4839-92d1-121c20f86f5f">